### ISSUE TITLE
Move series membership from Edition to Work

### ISSUE THREAD

**Issue by Minnozz** [bookwyrm-social/bookwyrm#3306]
The Wikidata property `part of the series` ([P179](https://www.wikidata.org/wiki/Property:P179)) links a `work` ([Q386724](https://www.wikidata.org/wiki/Q386724)) to a `series of creative works` ([Q7725310](https://www.wikidata.org/wiki/Q7725310)).

`work` is equivalent to our `Work` model. We currently store series information in the `Book` model, which is the parent class of both `Work` and `Edition`, but we don't expose `Work` to the end user: editing a "book" always edits an `Edition`.

I propose moving series membership from `Book` to `Work`, and making `Work` editable by the end user.

This would be a good time to introduce a `Series` model so the `series of creative works` Wikidata key can also be stored.

**Comment by hughrun** [comment]
I agree we need a `Series` model and that it should not link to `Book`.

Whether it should be linked to a `Work` or an `Edition` however depends on what we mean by "series", and this needs to be made very clear in all documentation and user interfaces to avoid confusion. A couple of examples to illustrate what I mean:

### Emma / Jane Austen

- _Everyman's Library_
- _Penguin Classics_
- _Vintage Classics Austen Series_
- etc

### The Light Fantastic / Terry Pratchett

- _Discworld_ (2)

Personally I would argue that in all the cases for _Emma_ these are **imprints** rather than a "series", and therefore in our small example above only _Discworld_ is a true series. But this is not necessarily obvious nor a universally held view (especially given that one of the imprints refers to itself as a "series"). The reason this matters is because imprints like "Penguin Classics" are only valid at an `Edition` level - the _Everyman's Library_ edition is not part of the _Penguin Classic_ "series". So how we define a series determines whether it links to `Edition` or `Work`, and vice-versa.
